core/src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -528,7 +540,7 @@ public void copyItemsetAnswer(QuestionDef q, TreeElement targetNode, IAnswerData
         triggerTriggerables(destRef);
 
         // initialize conditions for the node (and sub-nodes)
-        initializeTriggerables(destRef);
+        initTriggerablesRootedBy(destRef);
```

Comment:
@ctsims Can you give context on when this method (`copyItemsetAnswer`) is used? It is only called for complex questions in `FormEntryController.anserQuestion`. Supposedly complex questions don't really occur often? Is it okay that this call here is calling `fillTriggeredElements` with `isRepeatEntryInit` set to true? or should I isolate it?